### ISSUE TITLE
Ability to Change Specific GPU Overclock in Actions

### ISSUE THREAD

**Issue by XyKER1** [BartoszCichecki/LenovoLegionToolkit#1365]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar feature request mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.23.0

### OS

Windows 11 23H2 build 22631

### Device

Legion Pro 7i 16IRX8H

### Is your feature request related to a problem?

It would be nice to be able to change the GPU overclock in actions. Right now, all you can do is turn the overclock on and off. However, I would like to be able to set it to a specific value.
The reason for this is that most games I can run a very high overclock on my GPU, but in a few specific games, this will lead to crashing. I would like to use the "when app starts" and "when app closes" to automatically change my GPU overclock to an appropriate value to optimize my performance.

### How would you like the problem to be solved?

Add a step in the Actions tab called "set GPU overclock" that allows me to set a specific GPU overclock value and apply it.

### What alternatives have you considered?

Turning off the GPU overclock, but it would be nicer to just be able to lower the overclock instead. I know this would make a pretty minor difference, but I don't think it would be that hard to add.

**Comment by XyKER1** [comment]
@BartoszCichecki Any updates on this? If you don't want to do it, then I'm fine if you close this. I just want to make sure you saw it in the first place! :)

**Comment by BartoszCichecki** [comment]
Yes I did :) I will take a look at this at some point. I had very little time recently, so only most important things got pushed into release.

**Comment by XyKER1** [comment]
Closing this as I no longer care about this feature and don't use my Lenovo Legion much anymore after having so many issues with it.
